payloads/library/GitBunnyGit/payload.txt

```diff
@@ -18,19 +18,19 @@ LED R G B
 ATTACKMODE RNDIS_ETHERNET 
 #ATTACKMODE ECM_ETHERNET
 
-source bunny_helpers.sh
-
 # Set your desired repo url and branch if you're not looking to update from mainline
 git_repo="https://github.com/hak5/bashbunny-payloads.git"
 git_branch="master"
 payloads_dir="/root/udisk"
-log_file="/root/udisk/payloads/$SWITCH_POSITION/git.log"
+log_file="/root/udisk/git.log"
+
+echo " ---------  Git Bunny Git [started] -----------" >> $log_file
 
-echo "Git Bunny Git" > $log_file
+[[ ! -d "$payloads_dir/payloads" ]] && { LED R 200; echo "Could not mount filesystem" >> $log_file; exit 1; }
```

Comment:
I don't think this is a necessary check. Besides, if the clone ever fails and doesn't pull the `payloads` dir, the next time they try to run the payload, they will get this error, which isn't right.